controllers/replication-controller/dellcsireplicationgroup_controller_test.go

```diff
@@ -424,6 +425,36 @@ func (suite *RGControllerTestSuite) TestRGSyncDeletion() {
 	suite.Equal(false, resp.Requeue)
 }
 
+func (suite *RGControllerTestSuite) TestRGSyncWithNoDeletionTimeStamp() {
+
+	suite.createSCAndRG(suite.getTypicalSC(), suite.getRGWithoutSyncComplete(suite.driver.RGName, true, false))
+    rg := new(repv1.DellCSIReplicationGroup)
+    req := suite.getTypicalRequest()
+    err := suite.client.Get(context.Background(), req.NamespacedName, rg)
+    suite.NotContains(controllers.RGSyncComplete, rg.Finalizers,
+        "RG finalizer doesn't exist")
+    resp, err := suite.reconciler.Reconcile(context.Background(), req)
+    suite.NoError(err)
+    suite.Equal(false, resp.Requeue)
+    err = suite.client.Get(context.Background(), req.NamespacedName, rg)
+    suite.NoError(err)
+    rg.Finalizers = append(rg.Finalizers, controllers.RGFinalizer)
+	rg.DeletionTimestamp = &metav1.Time{
+		Time: time.Now(),
+	}
+    suite.T().Log(rg.Finalizers, "RG finalizer added")
+    // Check if remote RG got created
+    rClient, err := suite.config.GetConnection(suite.driver.RemoteClusterID)
+    suite.NoError(err)
+    _, err = rClient.GetReplicationGroup(context.Background(), rg.Name)
+    suite.NoError(err)
+    
+    //Delete rg
+    err = suite.client.Delete(context.Background(), rg)
```

Comment:
Could you explain what this test is for? When RG is deleted, there will be deletionTimeStamp set but the test name says 'noDeletionTimeStamp'.